packages/swap/src/helpers/orders/getFrontendFee.ts

```diff
@@ -0,0 +1,65 @@
+import {Balance} from '@yoroi/types'
+import BigNumber from 'bignumber.js'
+
+import {Quantities} from '../../utils/quantities'
+import {
+  DiscountTier,
+  milkHoldersDiscountTiers,
+} from '../../translators/constants'
+import {asQuantity} from '../../utils/asQuantity'
+
+export const getFrontendFee = ({
+  sellAmount,
+  buyAmount,
+  milkBalance,
+  sellInPrimaryTokenValue,
+  buyInPrimaryTokenValue,
+  primaryTokenInfo,
+  discountTiers = milkHoldersDiscountTiers,
+}: {
+  sellAmount: Balance.Amount
+  buyAmount: Balance.Amount
+  milkBalance: Balance.Quantity
+  sellInPrimaryTokenValue: Balance.Amount
```

Comment:
Do we get these from finding a pool between sell token and ada?

Reply:
these are received from the order + some metadata that it still don't have such as the sell/buy value in primary when they are not the primary. iUSD/DJED -> iUSD/ADA + DJED/ADA (based on what's selling)